patchwork-capabilities/src/main/java/net/patchworkmc/api/capability/CapabilityRegisteredCallback.java

```diff
@@ -29,6 +29,10 @@
 	void onCapabilityRegistered(Capability<C> capability);
 
 	static <C> Event<CapabilityRegisteredCallback<C>> event(Class<C> type) {
-		return CapabilityRegisteredCallbackInternal.getOrCreateEvent(type);
+		return CapabilityRegisteredCallbackInternal.getOrCreateEvent(type.getName());
+	}
+
+	static <C> Event<CapabilityRegisteredCallback<C>> event(String className) {
+		return CapabilityRegisteredCallbackInternal.getOrCreateEvent(className);
```

Comment:
I'm not confident we should allow arbitrary strings, but i guess in the long run it's really not that big of a deal